packages/safety-suite-api/src/api/permission/grants.ts

```diff
@@ -15,7 +33,7 @@ export const getGrant = (
   userId: Id,
   query: CustomerSpecificQuery,
   apiOptions?: ApiOptions
-): Request<ApiResponse<GrantDTO>> =>
+): Request<ApiResponse<GrantDTO[]>> =>
```

Comment:
We should update the name of this function to pluralize grant and update the JSDoc to reflect the fact that this is fetching all grants for the specified customer.